CONTRIBUTING.md

```diff
@@ -6,23 +6,23 @@ welcome to contribute towards development in the form of peer review, documentat
 testing and patches.
 
 Anyone is invited to contribute without regard to technical experience, "expertise", OSS
-experience, age, or other concern. Though developing cryptocurrencies demand a
+experience, age, or other concern. However, developing cryptocurrencies demand a
 high-level of rigor, adversial thinking, thorough testing and risk-minimization.
-Any bug may cost users real money. That said we're deeply welcoming people contributing
+Any bug may cost users real money. That said we deeply welcome people contributing
 for the first time to an open source project or pick up Rust while contributing. Don't be shy,
 you'll learn.
 
 Communications Channels
 -----------------------
 
-Communication about Rust-Lightning happens primarily on #ldk-dev on the LDK slack
+Communication about Rust-Lightning happens primarily on #ldk-dev on the [LDK slack](http://www.lightningdevkit.org/)
```

Comment:
if it's primarily on Slack, there should at least be a comma before the IRC channel, or, better yet, a connection like, "but also"

Reply:
fixed